.github/workflows/analyze.yml

```diff
@@ -0,0 +1,39 @@
+name: CI Static Analysis
+on: [push, pull_request]
+jobs:
+  gcc:
```

Comment:
Name to GCC and Clang casing to be consistent with job names in other workflows..?

Reply:
Renamed and added versions to be more clear.